extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php

```diff
@@ -33,67 +39,67 @@ public function __construct( User $user ) {
 	}
 
 	/**
-	 * Gets top wikis from DB for devboxes from method UserIdentityBox::getTestData()
+	 * Gets top wikis from stats DB based on number of edits
 	 *
 	 * @param Integer|null $limit
 	 *
 	 * @return array
 	 */
-	private function getTopWikisFromDb( $limit = null ) {
-		global $wgCityId, $wgDevelEnvironment, $wgSpecialsDB;
-
-		if ( is_null( $limit ) ) {
-			$limit = self::MAX_FAV_WIKIS;
-		}
+	private function getTopWikisFromDb( $limit = self::MAX_FAV_WIKIS ) {
+		global $wgCityId, $wgSpecialsDB;
 
 		if ( empty( $wgCityId ) ) {
 			// staff/internal does not have stats db
 			$wikis = [];
-		} else if ( $wgDevelEnvironment ) {
```

Comment:
They were using $wgDevelEnvironment as a way to cook the data in dev for what was essentially guided unit tests.  The `getTestData` method returned a set of preselected wikis, one of which was one that should be filtered.  This really belongs in a unit test somehwere.  Also, there wasn't any "strange behavior" that I could tell, so I thought it was best to get rid of this fake code path.